### DISCUSSION TITLE
2021-05-03 Meetup

### DISCUSSION THREAD

**Discussion by lesserwhirls** [Unidata/netcdf-java#694] [NetCDF-Java Meetups]
Topic: Version 7 planning

[Notes](https://docs.google.com/document/d/11KR_RawvurC3chCcziwARCOWo_aDgDTWv7D6NzJUc0I/edit#heading=h.114uf7f6otjd) (Google Doc - publicly viewable)

**Comment by JohnLCaron** [comment]
added some notes and comments
